docs-ref-conceptual/spring-framework/configure-spring-boot-java-applicationinsights.md

```diff
@@ -0,0 +1,314 @@
+        <artifactId>spring-boot-starter-web</artifactId>
+        <exclusions>
+            <exclusion>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-logging</artifactId>
+            </exclusion>
+        </exclusions>
+    </dependency>
+
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-test</artifactId>
+        <scope>test</scope>
+    </dependency>
+
+    <dependency>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-log4j2</artifactId>
+    </dependency>
+
+    <dependency>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>applicationinsights-spring-boot-starter</artifactId>
+        <version>1.0.0-BETA</version>
+    </dependency>
+
+    <dependency>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>applicationinsights-logging-log4j2</artifactId>
+        <version>2.1.1</version>
+    </dependency>
+</dependencies>
+
+```
+
+2. Save and close the *POM.xml* file.
+
+3. In \src\main\resources folder, create a new file *log4j2.xml* and configure it. For example:
+
+```xml
+<?xml version="1.0" encoding="UTF-8" ?>
+<Configuration packages="com.microsoft.applicationinsights.log4j.v2">
+  <Properties>
+    <Property name="LOG_PATTERN">
+      %d{yyyy-MM-dd HH:mm:ss.SSS} %5p ${hostName} --- [%15.15t] %-40.40c{1.} : %m%n%ex
+    </Property>
+  </Properties>
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+    </Console>
+    <ApplicationInsightsAppender name="aiAppender">
+    </ApplicationInsightsAppender>
+  </Appenders>
+  <Loggers>
+    <Root level="trace">
+      <AppenderRef ref="Console"  />
+      <AppenderRef ref="aiAppender"  />
+    </Root>
+  </Loggers>
+</Configuration>
+```
+4. Build and run the spring boot application again as shown above. 
+
+Within few seconds, you should see all the spring logs being avaialble on Azure Portal. 
+
+![Azure Portal][AZ06]
+
+You can even look at the detailed log messages and do analysis on Analytics Portal. 
+
+![Azure Portal][AZ07]
+
+## Next steps
+
+For more information about using Spring Boot applications on Azure, see the following articles:
+
+* [Deploy a Spring Boot Application to the Azure App Service](deploy-spring-boot-java-web-app-on-azure.md)
+
+* [Running a Spring Boot Application on a Kubernetes Cluster in the Azure Container Service](deploy-spring-boot-java-app-on-kubernetes.md)
+
+Application Insights supports automatic collection of external dependencies and it's correlation with incoming requests. Currently we support autocollection of Oracle, MsSQL, MySQL and Redis. For more details on enabling autocollection please follow the article [how to use Application Insights Java agent](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-agent).
```

Comment:
Contraction versus possessive. Change it's to its.
it's = contraction for "it is"
its = possessive form